### PR TITLE
implement minification in in the makefile

### LINKED ISSUE #63: Não estamos mais servindo arquivos minificados
Por algum motivo, no _commit_ 9f5858fab436054e8c33839cd1e2fc17276cd72b os arquivos minificados foram apagados e os arquivos `.setup` estão configurados para servir os arquivos não minificados.

Como o _commit_ foi feito sem passar por processo de revisão, imagino que pode ter sido um engano (a pessoa pode ter desabilitado a minificação para debugar mais facilmente localmente e esqueceu de reverter). Não vejo motivo para servir os arquivos não minificados.

Proponho reverter essa parte do _commit_ e voltar a servir os arquivos minificados, o que deve melhorar o tempo de carregamento da página, mesmo que marginalmente.

### PR DESCRIPTION
**Pull request by mateusfccp**
Fixes #63 

This reduces the total size of the static assets from 77.39kB to 44.56kB (about 57% of the original size).

Also, as we are bundling together the CSS files, we reduce the number of requests by one, which should be marginally faster.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,44 +1,47 @@
...
-	      -not -path "./.github/*" \
-	      -exec cp {} static \;
-	@find . \
-	      -name "*.html" \
-	      -not -path "./static/*" \
-	      -not -path "./.github/*" \
-	      -exec mv {} static \;
+minify:
+	@printf $(STATUS_PREFIX); echo "MINIFYING AND BUNDLING CSS FILES"
+	@$(DOCKER_RUN) --entrypoint="" $(MINIFY_IMG) sh -c 'minify --bundle org-theme/dist/long/bundle.css css/syntax.css -o $(STATIC_FILES_FOLDER)/long.css'
```
Comment (ryukinix) [inline Makefile]: Why this empty entrypoint is necessary? Which is the default one?

File: org-theme/setup/long.setup [hunk 2]
```diff
@@ -20,6 +20,5 @@
 # I recommend you to generate your own css/js files by forking, cloning and customizing the file
 # theme.json before building it.
 #
-#+HTML_HEAD: <link rel="stylesheet" type="text/css" href="bundle.css"/>
-#+HTML_HEAD: <link rel="stylesheet" type="text/css" href="syntax.css" />
-#+HTML_HEAD: <script src="bundle.js" type="text/javascript" ></script>
+#+HTML_HEAD: <link rel="stylesheet" type="text/css" href="long.css"/>
```
Comment (ryukinix) [inline org-theme/setup/long.setup:23]: cool @mateusfccp

File: Makefile (hunk 1 above)
Comment (ryukinix) [reply Makefile]: I read the Dockerfile recipe from this image, they set a init.sh script which wraps minify binary.

Maybe a less convoluted option:

```suggestion
	@$(DOCKER_RUN) --entrypoint="/usr/bin/minify" $(MINIFY_IMG) --bundle org-theme/dist/long/bundle.css css/syntax.css -o $(STATIC_FILES_FOLDER)/long.css
```

or even:

```suggestion
	@$(DOCKER_RUN) $(MINIFY_IMG) --bundle org-theme/dist/long/bundle.css css/syntax.css -o $(STATIC_FILES_FOLDER)/long.css
```

**Review by ryukinix — CHANGES_REQUESTED** [review]
Thanks again for another contribution. I wrote a small suggestion, give a look.

File: Makefile (hunk 1 above)
Comment (mateusfccp) [reply Makefile]: I tried not passing an entry point, as suggested by the documentation. For some reason, I got a permission error with `init.sh`.

I didn't manage to solve it, but by calling `minify` directly the error didn't happen.

If you know another way to solve this permission error so we don't have to pass an entry point, please let me know.

Comment (mateusfccp) [reply Makefile]: I made a change based on your first suggestion.

**Review by ryukinix — APPROVED** [review]
LGTM

**Pull request merged**